pages/index.js

```diff
@@ -5,11 +5,13 @@ import UserAgentUtils from 'utils/userAgent'
 import parser from 'ua-parser-js';
 import { BaseContainer } from "component/Page/BaseContainer";
 import { Helmet } from 'react-helmet';
+import getConfig from 'next/config';
 
 export function getServerSideProps(context) {
+  const { publicRuntimeConfig: config } = getConfig();
   const { postId, postPrivateId, postExpired, communityName } = context?.query
   const userAgent = parser(context?.req?.headers['user-agent'])
-  console.log('communityName', communityName)
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `getConfig` and its usage to destructure `publicRuntimeConfig` is a good practice for accessing environment variables in Next.js. However, it's important to ensure that the `WIX_URL` variable is properly set in your `.env` file or in your hosting environment. If not, this could lead to unexpected behavior when redirecting.

```diff
- const { publicRuntimeConfig: config } = getConfig();
+ const { publicRuntimeConfig: config = {} } = getConfig();
```

This change ensures that if `getConfig()` returns `undefined`, `config` will default to an empty object, preventing potential errors when trying to access properties on `config`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->